src/components/Modal/EditSpace.vue

```diff
@@ -12,7 +12,8 @@ const DEFAULT_FORM_STATE: SpaceMetadata = {
   discord: '',
   votingPowerSymbol: '',
   walletNetwork: null,
-  walletAddress: null
+  walletAddress: null,
+  delegationApiUrl: ''
```

Comment:
Shouldn't this be an array, or how do you imagine supporting multiple networks? @bonustrack

Reply:
I imagine something like this for Snapshot: https://github.com/orgs/snapshot-labs/projects/11/views/7?pane=issue&itemId=27520397

Reply:
If a space has a delegation system on multiple network he would need an unified API that merge voting power, we shouldnt support multiple network here.

Reply:
Why are we not doing that also for our own delegation subgraphs? Instead we have all these:

```json
{
  "1": "https://gateway.thegraph.com/api/0f15b42bdeff7a063a4e1757d7e2f99e/deployments/id/QmXZiV6S13ha6QXq4dmaM3TB4CHcDxBMvGexSNu9Kc28EH",
  "5": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-goerli",
  "10": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-optimism",
  "56": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-binance-smart-chain",
  "100": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-gnosis-chain",
  "137": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-polygon",
  "250": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-fantom",
  "42161": "https://api.thegraph.com/subgraphs/name/snapshot-labs/snapshot-arbitrum"
}
```

Reply:
This is also very slow when loading on the Ui for some reason

Reply:
Our subgraph have only the mapping of delegation not the voting power, this is why its super slow

Reply:
> If a space has a delegation system on multiple network he would need an unified API that merge voting power, we shouldnt support multiple network here.

But should be support multiple URLs? Quote from the Delegation pitch "providing links to the relevant APIs", this does sound like it should support multiple URLs

Reply:
Once we have multiple standards, it would also be beneficial to consider supporting them simultaneously.

Reply:
If a user clicks delegate on the Ui, how does the Ui know on which network the delegation should happen by the way?